### ISSUE TITLE
pip install fails on python 3.7, 3.8, 3.9, 3.10

### ISSUE THREAD

**Issue by caroheymes** [Nykakin/chompjs#48]
I've tried to install chompjs on windows with pip but install fails.
Here is the trace
`Collecting chompjs
  Using cached chompjs-1.2.0.tar.gz (17 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: chompjs
  Building wheel for chompjs (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [29 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-37
      creating build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\chompjs.py -> build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\test_parser.py -> build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\__init__.py -> build\lib.win-amd64-cpython-37\chompjs
      running egg_info
      writing chompjs.egg-info\PKG-INFO
      writing dependency_links to chompjs.egg-info\dependency_links.txt
      writing top-level names to chompjs.egg-info\top_level.txt
      reading manifest file 'chompjs.egg-info\SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE'
      writing manifest file 'chompjs.egg-info\SOURCES.txt'
      running build_ext
      building '_chompjs' extension
      creating build\temp.win-amd64-cpython-37
      creating build\temp.win-amd64-cpython-37\Release
      creating build\temp.win-amd64-cpython-37\Release\_chompjs
      "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\cheymes\Anaconda3\envs\chomp\include -IC:\Users\cheymes\Anaconda3\envs\chomp\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tc_chompjs/buffer.c /Fobuild\temp.win-amd64-cpython-37\Release\_chompjs/buffer.obj
      buffer.c
      _chompjs/buffer.c(45): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
      "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\cheymes\Anaconda3\envs\chomp\include -IC:\Users\cheymes\Anaconda3\envs\chomp\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tc_chompjs/module.c /Fobuild\temp.win-amd64-cpython-37\Release\_chompjs/module.obj
      module.c
      _chompjs/module.c(88): error C2099: l'initialiseur n'est pas une constante
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.35.32215\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for chompjs
  Running setup.py clean for chompjs
Failed to build chompjs
ERROR: Could not build wheels for chompjs, which is required to install pyproject.toml-based projects

(chomp) C:\WINDOWS\system32>pip install chompjs
Collecting chompjs
  Using cached chompjs-1.2.0.tar.gz (17 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: chompjs
  Building wheel for chompjs (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [29 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-37
      creating build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\chompjs.py -> build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\test_parser.py -> build\lib.win-amd64-cpython-37\chompjs
      copying chompjs\__init__.py -> build\lib.win-amd64-cpython-37\chompjs
      running egg_info
      writing chompjs.egg-info\PKG-INFO
      writing dependency_links to chompjs.egg-info\dependency_links.txt
      writing top-level names to chompjs.egg-info\top_level.txt
      reading manifest file 'chompjs.egg-info\SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE'
      writing manifest file 'chompjs.egg-info\SOURCES.txt'
      running build_ext
      building '_chompjs' extension
      creating build\temp.win-amd64-cpython-37
      creating build\temp.win-amd64-cpython-37\Release
      creating build\temp.win-amd64-cpython-37\Release\_chompjs
      "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\cheymes\Anaconda3\envs\chomp\include -IC:\Users\cheymes\Anaconda3\envs\chomp\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tc_chompjs/buffer.c /Fobuild\temp.win-amd64-cpython-37\Release\_chompjs/buffer.obj
      buffer.c
      _chompjs/buffer.c(45): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
      "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\cheymes\Anaconda3\envs\chomp\include -IC:\Users\cheymes\Anaconda3\envs\chomp\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.35.32215\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tc_chompjs/module.c /Fobuild\temp.win-amd64-cpython-37\Release\_chompjs/module.obj
      module.c
      _chompjs/module.c(88): error C2099: l'initialiseur n'est pas une constante
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.35.32215\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for chompjs
  Running setup.py clean for chompjs
Failed to build chompjs
ERROR: Could not build wheels for chompjs, which is required to install pyproject.toml-based projects`

Thanks a lot for your help

**Comment by Nykakin** [comment]
I believe this should be resolved by https://github.com/Nykakin/chompjs/pull/49 but I don't have a Windows to check it. Uploaded the version 1.2.1 to TestPyPI. Please try to verify that everything works with
```
python3 -m pip install --index-url https://test.pypi.org/simple/ chompjs
```
If it works I'll deploy 1.2.1 to the main PyPI.

**Comment by Nykakin** [comment]
Managed to test it on Windows machine, looks like it's resolved with 1.2.1.
